platform/core/src/schema.js

```diff
@@ -1,34 +0,0 @@
-//import SimpleSchema from 'simpl-schema'
```

Comment:
We don't use this anymore, but having something similar as our config continues to grow might be nice. Curious if there's a good FOSS package that handles simple schema validation like this. I guess, in a way, react `prop-types` already do.